server/controllers/project.controller.js

```diff
@@ -89,14 +89,28 @@ export function getProjects(req, res) {
 
 export function getProjectsForUser(req, res) {
   if (req.params.username) {
-    User.findOne({ username: req.params.username }, (err, user) => {
-      Project.find({user: user._id}) // eslint-disable-line no-underscore-dangle
+    // Should we return all sketches when the username is p5? Maybe just public ones?
+    if (req.params.username === 'p5') {
```

Comment:
In production, the idea is that the "p5" user will have a bunch of projects intended to be examples (#149). So we don't need this 😄